optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -220,12 +225,44 @@ public void writeReport() {
         for (ProblemBenchmarkResult problemBenchmarkResult : plannerBenchmarkResult.getUnifiedProblemBenchmarkResultList()) {
             if (problemBenchmarkResult.hasAnySuccess()) {
                 for (ProblemStatistic problemStatistic : problemBenchmarkResult.getProblemStatisticList()) {
+                    for (SingleStatistic singleStatistic : problemStatistic.getSingleStatisticList()) {
+                        try {
+                            singleStatistic.unhibernatePointList();
+                        } catch (IllegalStateException ise) {
```

Comment:
The default variable name for an exception is e.
There a few these very short default names for which it's widely accepted not to give a good meaningful name:
for (int i = 0;...)
catch (...Exception e)